src/Microsoft.VisualBasic.Core/tests/InteractionTests.cs

```diff
@@ -98,5 +114,51 @@ private static IEnumerable<object[]> IIf_TestData()
             yield return new object[] { false, 3, "str", "str" };
             yield return new object[] { true, 3, "str", 3 };
         }
+
+        [Theory]
+        [InlineData(0, 1, 2, 1, " :0")]
+        [InlineData(1, 1, 2, 1, "1:1")]
+        [InlineData(2, 1, 2, 1, "2:2")]
+        [InlineData(3, 1, 2, 1, "3: ")]
```

Comment:
>3, 1, 2, 1, "3: " [](start = 20, length = 17)

Perhaps `(10, 1, 9, 1, "10:  ")` as well since the width of "10" is greater than `Stop`.

Reply:
Added this